### ISSUE TITLE
Fail to install using Python 3

### ISSUE THREAD

**Issue by ratson** [edoburu/django-fluent-comments#62]
`akismet` is breaking the install on Python 3

```
$ pip install django-fluent-comments                                                                                                                                    node:system
Collecting django-fluent-comments
  Using cached django_fluent_comments-1.0.5-py2.py3-none-any.whl
Collecting akismet>=0.2 (from django-fluent-comments)
  Using cached akismet-0.2.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/private/var/folders/tp/d0cgdkdd1c32w57b4c_kl9d40000gn/T/pip-build-4au2m7mf/akismet/setup.py", line 5, in <module>
        import akismet
      File "/private/var/folders/tp/d0cgdkdd1c32w57b4c_kl9d40000gn/T/pip-build-4au2m7mf/akismet/akismet.py", line 136
        except Exception, e:
                        ^
    SyntaxError: invalid syntax

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/tp/d0cgdkdd1c32w57b4c_kl9d40000gn/T/pip-build-4au2m7mf/akismet
```

**Comment by kakai248** [comment]
What I did was checking the requirements and installing them one by one, except akismet. Then I installed django-fluent-comments with the option ```--no-deps```. You will be able to install it this way and if you don't use akismet everything will work perfectly.

**Comment by gustavklopp** [comment]
Thanks, it's working. 

What do you use in replacement for the anti-Spam Akismet (which is quite old: last version is from 2009!)

**Comment by kakai248** [comment]
I don't use any replacemenet. My comment system requires the user to be logged in. I also added a captcha.

**Comment by PetrDlouhy** [comment]
There is Python 3 compatible [version of Akismet](https://github.com/allieus/python-akismet.git). So you can add following line to your requirements file before django-fluent-comments:
```
-e git+https://github.com/allieus/python-akismet.git#egg=akismet
```

**Comment by vdboor** [comment]
Fixed in 1.1!

I've removed akismet as a fixed requirement in setup.py using conditional dependencies. If you install it, using the `requirements.txt` method above, it will be picked up and used.
